python/ceed.py

```diff
@@ -27,337 +27,341 @@
 from .ceed_constants import *
 
 # ------------------------------------------------------------------------------
+
+
```

Comment:
I'm not sure why we're getting these two blank lines after every single 
```
# ----------------
# description
# ----------------
```
comment. That feels like a linting error.

Reply:
I believe it's because our commenting style is discouraged by the PEP (which would prefer that be a docstring). Is it E265? https://pypi.org/project/autopep8/#features

Reply:
Block comments don't require two blank lines after them. I think this is what's causing the extra spacing "Surround top-level function and class definitions with two blank lines.".

Reply:
Now that I understand, I am ok with this. There are a couple of instances that look a little awkward, but most of it is adding padding around lines that look like
```
# ---------------
```
and those lines really help me find stuff in these files.